lessons/basics/comprehensions.md

```diff
@@ -105,7 +105,7 @@ iex> for {k, v} <- [one: 1, two: 2, three: 3], into: %{}, do: {k, v}
 %{one: 1, three: 3, two: 2}
 ```
 
-Since bitstrings are enumerable we can use list comprehensions and `:into` to create strings:
+Since binaries are collectables we can use list comprehensions and `:into` to create strings:
```

Comment:
Given that we're changing two important terms here it might make sense to bump the `minor` version on this lesson as well.